typing/gprinttyp.ml

```diff
@@ -742,10 +742,10 @@ module Digraph = struct
             fields
         in
         { elts; graph = add_subgraph (labelr "polyvar", fields) main }
-    | Types.Tpackage (p, fl) ->
+    | Types.Tpackage {pack_path; pack_cstrs = fl} ->
```

Comment:
Replacing the name `fl` which is used once would be nice.

Reply:
`fl` is used twice here. Replacing `fl` by `pack_cstrs` on line 750 would lead to going above 80 chars forcing to add a line break. Is this still better for presentation ?

Reply:
Going in that direction is indeed much nicer. Done